packages/utils/src/opensearch.ts

```diff
@@ -0,0 +1,50 @@
+import { Client } from "@opensearch-project/opensearch";
```

Comment:
was this file intentionally committed? if so, let's change the name to reflect what it actually does, and a comment at the top of the file

Reply:
Yeah, this is just a utility file so we have a ready-to-use script to interact with an OpenSearch instance. I'm updating this on the upcoming PR, will add the message at the top.